user-bag-ui/src/modules/client/src/components/create-user-form/index.tsx

```diff
@@ -0,0 +1,125 @@
+                    onSuccess();
+                })
+                .catch((error) => AppToaster.show({message: `Error creating account: ${error}`, intent: 'danger'}))
+        } catch(err) {
+            AppToaster.show({message: err, intent: "danger"})
+        }
+    }
+
+    return (
+        <div>
+            <FormGroup>
+                <label htmlFor="id">ID </label>
+                <InputGroup
+                    id="id"
+                    name="id"
+                    type="text"
+                    large
+                    onChange={(event: ChangeEvent<HTMLInputElement>) => setField(event, setId)}
+                />
+                <label htmlFor="name">Name </label>
+                <InputGroup
+                    id="name"
+                    name="name"
+                    type="text"
+                    large
+                    onChange={(event: ChangeEvent<HTMLInputElement>) => setField(event, setName)}
+                />
+                <label htmlFor="deposit">Initial Deposit </label>
+                <InputGroup
+                    id="deposit"
+                    name="deposit"
+                    type="number"
+                    large
+                    min={0}
+                    onChange={(event: ChangeEvent<HTMLInputElement>) => setField(event, setBalance)}
+                />
+                <label htmlFor="email">Email Address </label>
+                <InputGroup
+                    id="email"
+                    name="email"
+                    type="text"
+                    large
+                    onChange={(event: ChangeEvent<HTMLInputElement>) => setField(event, setEmail)}
+                />
+                <label htmlFor="phone">Phone Number </label>
+                <InputGroup
+                    id="phone"
+                    name="phone"
+                    type="text"
+                    large
+                    onChange={(event: ChangeEvent<HTMLInputElement>) => setField(event, setPhone)}
+                />
+                <label htmlFor="password">Password </label>
+                <InputGroup
+                    id="password"
+                    name="password"
+                    type="password"
+                    large
+                    onChange={(event: ChangeEvent<HTMLInputElement>) => setField(event, setPassword)}
+                />
+                <label htmlFor="confirmPassword">Confirm Password</label>
+                <InputGroup
+                    id="confirmPassword"
+                    name="confirmPassword"
+                    type="password"
+                    large
+                    onChange={(event: ChangeEvent<HTMLInputElement>) => setField(event, setConfirmPassword)}
+                />
+            </FormGroup>
+            <div>
+                <Button
+                    data-testid="create-user-submit-button"
+                    onClick={postToApi}
```

Comment:
If I click this multiple times, does it create multiple users?

Might want to add some more disabled logic to stop you doing that, especially whilst its loading (upto you)